### DISCUSSION TITLE
Directly go to Organization after login

### DISCUSSION THREAD

**Discussion by ahassan-lc** [community/community#58524] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

I have created a GitHub account and joined an organization
when i logged in it takes me to my personal account and then I have to choose the organization 
but I want when organization member login to their GitHub account they directly go into the organization

**Comment by the-officialjosh** [comment]
~~To configure your GitHub account to directly log in to your organization's account, you can set the organization as your primary login account. Here's how you can do it:~~

~~1. Log in to your GitHub account.
2. Go to your profile by clicking on your profile picture in the top right corner of the page.
3. In the drop-down menu, click on "Settings."
4. In the left sidebar, click on "Account" or "Account Settings."
5. Scroll down to the "Profile" section.
6. Under the "Organizations" heading, you will see a list of organizations you are a member of.
7. Find the organization you want to set as your primary login and click on it.
8. On the organization's page, you should see an option to "Set as default login."
9. Click on "Set as default login" or similar wording to set the organization as your primary login.~~

~~After setting the organization as your default login, whenever you log in to GitHub, you should be directly taken to your organization's account instead of your personal account.~~

~~Please note that this configuration is specific to your account, and each organization member would need to follow these steps individually to set the organization as their default login.~~

Oh, I am so sorry I just realised that that earlier response is wrong. I got it from another issue and it seems that the response is wrong yet it was marked as the answer however, it appears that this is not possible. You currently have to log in to your personal account first and cannot access organizations directly. I just saw this in the [GitHub Docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/accessing-an-organization)

**Reply by ahassan-lc** [reply]
Is there any way to set this for all user in the Organization and an admin of the organization.

**Reply by ahassan-lc** [reply]
![github](https://github.com/community/community/assets/132251620/fd1259d2-7939-4e36-a66a-4fc31beb11ab)

**Reply by ahassan-lc** [reply]
this page I'm getting this if I follow the steps you have shared. Can you have a look and let me know how to proceed?

**Reply by ahassan-lc** [reply]
I'm trying to follow the steps you have shared in the first place. I followed till step 4 and for step 5 I'm not getting the options as mentioned
I have shared the SS for reference so that you can let me know how to follow step 5 and onwards.

**Reply by the-officialjosh** [reply]
Oh, I am so sorry I just realised that that earlier response is wrong. I got it from another issue and it seems that the response is wrong yet it was marked as the answer however, it appears that this is not possible. You currently have to log in to your personal account first and cannot access organizations directly. I just saw this in the [GitHub Docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/accessing-an-organization)

**Reply by ahassan-lc** [reply]
No issues
Thanks for the update. Appreciate it.

**Reply by the-officialjosh** [reply]
You can close this particular discussion and mark that answer as the answer, I need it for my stats 😔
